mne/viz/evoked.py

```diff
@@ -1191,3 +1191,350 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     # show and return it
     plt_show(show)
     return fig
+
+
+def _ci(arr, ci):
+    """Calculate the `ci`% parametric confidence interval for `arr`.
+    Aux function for plot_compare_evokeds."""
+    from scipy import stats
+    mean, sigma = arr.mean(0), stats.sem(arr, 0)
+    # This is highly convoluted to support 17th century Scipy
+    # XXX Fix when Python 2.6 support is dropped!
```

Comment:
@agramfort can we drop 2.6 yet? I seem to recall us talking about dropping support for ancient NumPy or 2.6 next time we hit an issue with it (can't remember if it was one of those or both)